src/box/index.cc

```diff
@@ -959,6 +962,24 @@ index_read_view_delete(struct index_read_view *rv)
 	index_def_delete(def);
 }
 
+#ifndef NDEBUG
+
+int
+index_inject_oom(void)
+{
+	struct errinj *e = errinj(ERRINJ_INDEX_OOM, ERRINJ_BOOL);
+	ERROR_INJECT_COUNTDOWN(ERRINJ_INDEX_OOM_COUNTDOWN, e->bparam = true);
+	if (e->bparam && !txn_has_flag(in_txn(), TXN_STMT_ROLLBACK) &&
+	    in_txn()->status != TXN_ABORTED) {
+		diag_set(OutOfMemory, 777,
+			 "errinj", "errinj");
```

Comment:
Could be written in one line. Also, let's use 0 instead of 777 :)

Reply:
Fixed as part of another conversation.